k8s/Makefile

```diff
@@ -159,6 +159,15 @@ k8s-jenkins-create-rbac:
 k8s-jenkins-delete-rbac:
 	kubectl delete -f jenkins-rbac.yaml
 
+k8s-search-index-update-job: k8s-delete-search-index-update-job
+	env NEW_RELIC_APP_NAME=dev-portal-web-${TARGET_ENVIRONMENT} \
+	j2 search-index-update-job.yaml.j2 | ${KC} apply -f -
+	env JOB_NAME=search-index-update ./wait_for_job.sh
```

Comment:
I set a value for NEW_RELIC_APP_NAME here with the assumption that we'll get output from the command then, rather than it going to the equivalent of `/dev/null` -- is that right @limed ?

Reply:
Setting the name here just sets  the name of the app (in this case `dev-portal-web-${stage,prod}`) under APM in NR.